### ISSUE TITLE
Artist name and Track name is not correct

### ISSUE THREAD

**Issue by merovic** [fethica/FRadioPlayer#46]
arabic names and its printed like this "ÍÓíä ÇáÏíß"

**Comment by fethica** [comment]
@merovic can you share the stream that's returning that, thanks!

**Comment by merovic** [comment]
func radioPlayer(_ player: FRadioPlayer, metadataDidChange artistName: String?, trackName: String?) {
        
        print(artistName ?? "artistName")
        print(trackName ?? "trackName")

    }

**Comment by merovic** [comment]
I also tried to decode the text and fix the arabic issue but without any success

**Comment by fethica** [comment]
Yes, can you share the stream URL that is returning the arabic text to run a quick test?

**Comment by merovic** [comment]
https://c26.radioboss.fm:18110/stream.mp3
